### DISCUSSION TITLE
What are you doing with FreeRouting?

### DISCUSSION THREAD

**Discussion by lynchaj** [freerouting/freerouting#59] [Show and tell]
Hi
Thanks to the devs we now have a forum for FreeRouting.  Yay!  First post!

I've been using FreeRouting for several years to design retrobrewcomputer projects.  My current project is the Z80 Multi Board Computer (modular backplane computer)

https://www.retrobrewcomputers.org/forum/index.php?t=msg&th=568&start=0&

https://www.retrobrewcomputers.org/doku.php?id=builderpages:lynchaj:start

What are you doing with FreeRouting?  Post about your projects here and introduce yourself

Thanks, Andrew Lynch

PS, Using FreeRouting 1.2.43 Extra on Windows 10, 32 bit with KiCAD 5.1.10

**Comment by andrasfuchs** [comment]
I used freerouting for a few of my prototypes of my project called [Bio Balance Detector](https://github.com/andrasfuchs/BioBalanceDetector). These prototypes (like [Venus 256](https://github.com/andrasfuchs/BioBalanceDetector/wiki/Proto-%236---Venus-256), [Mars 64](https://github.com/andrasfuchs/BioBalanceDetector/wiki/Proto-%237---Mars-64) and [The Sleepy Bodyguard](https://github.com/andrasfuchs/BioBalanceDetector/wiki/Proto-%238-The-Sleepy-Bodyguard)) are data acquisition systems that provide a bio-potential reading in various frequency ranges. 

Their goal is to detect the miniature electromagnetic field changes around the human body to potentially detect anomalies that could indicate health problems in early stages.
